### PR TITLE
feat(widget_versioning): created widget_versioning and example

### PR DESCRIPTION
**Pull request by JPugetGil**
created:
- versioning widget
- associated example
- documentation

### CODE REVIEW

File: examples/widget_sparql.html [hunk 1]
```diff
@@ -19,11 +19,6 @@
      * The class that adds the display of a drop-down menu containing clustering functionalities when a node is clicked
      */
     class ContextMenuGraphClustering {
-      /**
-       * Create a new Context Menu
-       *
```
Comment (mathieuLivebardon) [inline examples/widget_sparql.html:24]: Why did you delete this part of the commentary?

File: examples/widget_versioning.html [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    <title>widget versioning</title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./assets/css/examples.css" />
+    <link rel="stylesheet" href="./assets/css/loading_screen.css" />
+    <link rel="stylesheet" href="./assets/css/widget_versioning.css" />
+  </head>
+  <body>
+    <script src="../dist/RUN_MODE/bundle.js"></script>
+    <script src="./assets/js/loadingScreen.js"></script>
```
Comment (mathieuLivebardon) [inline examples/widget_versioning.html:13]: You don't use this script you should call it or remove it

**Review by mathieuLivebardon — APPROVED** [review]
DiekatzleiDieKatzleiGIF

File: examples/widget_sparql.html (hunk 1 above)
Comment (JPugetGil) [reply examples/widget_sparql.html:24]: Typing in HTML is not a relevant thing: it produces an error

File: examples/widget_versioning.html (hunk 2 above)
Comment (JPugetGil) [reply examples/widget_versioning.html:13]: I fixed it, thanks !
